src/dcqc/tests/tests.py

```diff
@@ -158,3 +158,27 @@ def generate_process(self) -> Process:
             command_args=command_args,
         )
         return process
+
+
+class TiffTag306DateTimeTest(ExternalTestMixin, TestABC):
+    tier = 4
+
+    def generate_process(self) -> Process:
+        file = self.get_file()
+        path = file.local_path.as_posix()
+        command_args = [
+            "!",  # negate exit status
+            "tifftools",
+            "dump" "|" "grep",  # pipe the output
+            "-E",  # extended regular expression
+            "-i",  # case insensitive
+            "-a",  # treat input as text
+            "-q",  # suppress output
+            r"'DateTime 306 \(0x132\) ASCII'",  # match the DateTime 306 tag
```

Comment:
I needed to make this a regex string for the pre-commit to pass. I think that this is OK given that its not actually a regex string but rather just an list being pased to `Process`. Be good to confirm.